src/mn_setup.c

```diff
@@ -2488,20 +2500,18 @@ static setup_menu_t gen_settings4[] = {
     {"Camera Deadzone", S_THERMO | S_PCT, CNTR_X, M_THRM_SPC,
      {"joy_camera_inner_deadzone"}, .action = I_ResetGamepad},
 
+    {"Rumble", S_THERMO, CNTR_X, M_THRM_SPC, {"joy_rumble"},
```

Comment:
How about adding `MI_GAP` before “Rumble” to make it more noticeable?

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1876/commits/a89d66818f72daa1979b3336a88298c5800cefa6

I've been trying to keep the items above the status bar height for legibility, so I went with this:

![woof0000](https://github.com/user-attachments/assets/10f48ee1-a9db-487f-a138-3c9814f3ed56)

I'll probably change this in the future since I still want to move gyro to a sub-menu under the "Gamepad" tab.